src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/MissingOverride/context/expected.txt

```diff
@@ -0,0 +1,3 @@
+Test.java:12:5: The Javadoc {@inheritDoc} tag is not valid at this location.
+Test.java:6:5: Must include @java.lang.Override annotation when {@inheritDoc} Javadoc tag exists.
+Test.java:9:5: Must include @java.lang.Override annotation when {@inheritDoc} Javadoc tag exists.
```

Comment:
this violation should not pass, as path does not change this line at all.

`neverSuppressedChecks`  is for `never` if code is changed.

Reply:
@romani, context is to find and get which violation's lines are changed, if it can not distinguish some violations which should be passed, then we think context strategy failed, and then we use `neverSuppressedChecks`, `neverSuppressedChecks` just pass all violations in the file.

Reply:
Looks like we should have one more property. 

`neverSuppressedChecks` - just pass violation no matter what

`noSuppressionChecksIfInPatchScope` (or something like this) to let user configure here MissingOverride or some other "missing" Check.

Reply:
@HuGanghui , please create in this PR one more test folder for behavior without `neverSuppressedChecks`. To let us see behavior in different cases.

Reply:
done, I try it in `context/case2`,  then we only can capture violation `Test.java:6:5: Must include @java.lang.Override annotation when {@inheritDoc} Javadoc tag exists.` according to deleted line information, and others will be suppressed.

Reply:
@romani `noSuppressionChecksIfInPatchScope`, please give more details, I am not very clear.